### ISSUE TITLE
Scene.drillPick messed up on entities borders

### ISSUE THREAD

**Issue by sir-chaos** [AnalyticalGraphicsInc/cesium#6361]
tl;dr: it would be great to add picking rectangle height and width parameters to Scene.drillPick to pass to underlying Scene.pick.

There were some threads/issues about pick/drillPick messing up pickPosition. This one is similiar too, although its more about unlucky missing feature or two, then plain bug in Cesium source. Imagine a scene where you need to intensively use pickPosition to get under-cursor coordinates. Also, you have to turn off depth test both for translucent objects and globe (lets take that as given). Its OK that you pick through  translucent ones, but result for globe becomes a trouble, so if you end up pointing at globe, you want to use camera.pickEllipsoid instead of scene.pickPosition.

Unfortunately, pick and drillPick get translucent objects even with depth off, but pickPosition will pick through them, meaning that you effectively may get one object via pick and the other via pickPosition (or even not get any position at all, despite you were sure there is an object). To fix it, quite simply, I go through drillPick, and look if there is any non translucent object, if yes, use pickPosition (now I'm sure it will end up at that object). What made me puzzled, is that close to the edges of non-translucent geometries I got wrong results (i.e. result of using pickPosition on the globe). It took me some time before I understood that it isn't due to pick messing up pickPosition (as discussed in some threads, probably already fixed), but just the default rectangle for picking is not 1x1 but 3x3! So pick and pickPosition just legally got different results.

What is worse, it isn't really documented that drillPick is merely calling pick several times (with default width/height = 3x3), and lack of height/width parameters can easily confuse one to think it would pick 1x1 by default. And even knowing that, only way for me to make drillPick work consistently with pickPosition is to patch Cesium.

Thus, I think it would be great to add the aforementioned parameters to drillPick(and just pass them to underlying pick, must be easy). Other options could do too, like making drill pick 1x1, but I believe such hardcoding has its own downsides. Concluding, I believe that having all means of picking consistent is rather important and it would be great to make this achievable.

**Comment by hpinkos** [comment]
Thanks @sir-chaos, this seems like a good idea to me!  If you have time to put together a pull request we'd be happy to review it =)

**Comment by sir-chaos** [comment]
Sure, I'll try to do that soon.

**Comment by sir-chaos** [comment]
@hpinkos added pull request https://github.com/AnalyticalGraphicsInc/cesium/compare/master...sir-chaos:patch-2

**Comment by Futhark** [comment]
this should be closed since these changes are already implemented
